docker/oap-es7/docker-entrypoint.sh

```diff
@@ -384,8 +384,6 @@ receiver-jvm:
   default:
 receiver-clr:
   default:
-receiver-so11y:
```

Comment:
should move it to func `generateTelemetrySo11y `

Reply:
It's already there, isn't it?

https://github.com/apache/skywalking/blob/5eba81e1409ca11204812bd456d4f4af206b76d2/docker/oap/docker-entrypoint.sh#L450-L454

Reply:
sorry, didn't notice it